src/Illuminate/View/ViewException.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Illuminate\View;
+
+use ErrorException;
+
+class ViewException extends ErrorException
+{
+    /**
+     * Report the exception.
+     *
+     * @return void
+     */
+    public function report()
+    {
+        $exception = $this->getPrevious();
+
+        if ($exception && method_exists('report', $exception)) {
```

Comment:
The `method_exists()` arguments seem to be in the wrong order? https://www.php.net/method_exists

The 8.x branch will need to be fixed too.

Reply:
Aaah, I fixed it for render but forgot for report. Thanks, I'll see to it that it gets fixed on 8.x as well.